<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Vivek-Deevis-MacBook-Pro.local" ignored="0" name="Default test" tests="3" failures="1" timestamp="13 Mar 2022 13:13:53 GMT" time="86.032" errors="0">
  <testcase name="feature" time="43.404" classname="com.ahb.mobileautomation.runner.RunCukesTest"/>
  <testcase name="feature" time="18.475" classname="com.ahb.mobileautomation.runner.RunCukesTest"/>
  <testcase name="feature" time="23.524" classname="com.ahb.mobileautomation.runner.RunCukesTest">
    <failure type="cucumber.runtime.CucumberException" message="junit.framework.AssertionFailedError: Both actual and expected are mismatched expected:&amp;lt;600522228&amp;gt; but was:&amp;lt;600522229&amp;gt;">
      <![CDATA[cucumber.runtime.CucumberException: junit.framework.AssertionFailedError: Both actual and expected are mismatched expected:<600522228> but was:<600522229>
at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
at com.ahb.mobileautomation.runner.RunCukesTest.feature(RunCukesTest.java:55)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: junit.framework.AssertionFailedError: Both actual and expected are mismatched expected:<600522228> but was:<600522229>
at junit.framework.Assert.fail(Assert.java:50)
at junit.framework.Assert.failNotEquals(Assert.java:287)
at junit.framework.Assert.assertEquals(Assert.java:67)
at junit.framework.Assert.assertEquals(Assert.java:199)
at com.ahb.mobileautomation.pages.MenuPage.verifyAutoPopulatedNumber(MenuPage.java:97)
at com.ahb.mobileautomation.stepdefinitions.MainSteps.check_if_is_auto_populated_in_the_dialer_screen_or_user_will_get_option_to_call_the_same_number(MainSteps.java:101)
at ✽.Then Check if ‘600522228’ is auto-populated in the dialer screen or user will get option to call the same number(3Callus.feature:18)
]]>
    </failure>
  </testcase> <!-- feature -->
</testsuite> <!-- Default test -->
